src/common/platform_win32.cpp

```
@@ -0,0 +1,40 @@
+ * Selected by the Makefile when OS == Windows_NT.
+ * @see ADR-170
+ */
+#include "platform.h"
+
+#include <string>
+#include <windows.h>
+
+namespace platform {
+
+std::string executable_path() {
+  char buf[MAX_PATH] = "";
```

Clang (14.0.6)

[warning] variable 'buf' is not initialized

(cppcoreguidelines-init-variables)